src/Wpf.Ui/Controls/ListView/ListView.xaml

```diff
@@ -145,14 +142,21 @@
         <Setter Property="ScrollViewer.CanContentScroll" Value="True" />
         <Setter Property="VerticalContentAlignment" Value="Center" />
         <Setter Property="VirtualizingPanel.IsVirtualizing" Value="True" />
+        <Setter Property="VirtualizingPanel.IsVirtualizingWhenGrouping" Value="True" />
         <Setter Property="VirtualizingPanel.VirtualizationMode" Value="Standard" />
         <Setter Property="SnapsToDevicePixels" Value="True" />
         <Style.Triggers>
             <DataTrigger Binding="{Binding Path=ViewState, RelativeSource={RelativeSource Mode=Self}}" Value="{x:Static controls:ListViewViewState.Default}">
                 <Setter Property="ItemsPanel">
                     <Setter.Value>
                         <ItemsPanelTemplate>
-                            <VirtualizingStackPanel IsVirtualizing="{TemplateBinding VirtualizingPanel.IsVirtualizing}" VirtualizationMode="{TemplateBinding VirtualizingPanel.VirtualizationMode}" />
+                            <VirtualizingStackPanel
+                                CacheLength="{TemplateBinding VirtualizingPanel.CacheLength}"
+                                CacheLengthUnit="{TemplateBinding VirtualizingPanel.CacheLengthUnit}"
+                                IsVirtualizing="{TemplateBinding VirtualizingPanel.IsVirtualizing}"
+                                IsVirtualizingWhenGrouping="{TemplateBinding VirtualizingPanel.IsVirtualizing}"
+                                ScrollUnit="{TemplateBinding VirtualizingPanel.ScrollUnit}"
+                                VirtualizationMode="{TemplateBinding VirtualizingPanel.VirtualizationMode}" />
```

Comment:
Allows the user to override the default behavior, such as when grouping is enabled.